Source/Urho3D/Graphics/Terrain.cpp

```diff
@@ -524,6 +525,10 @@ void Terrain::SetOccludee(bool enable)
 
     MarkNetworkUpdate();
 }
+void Terrain::SetEnableDebug(bool enable)
+{
+    debugGeometry_ = enable;

```

Comment:
Is it needed to dirty anything?

Reply:
This should be called before creating any terrain. All this flag does is make `TerrainPatch` vertex buffers shadowed upon (re)creation. So i guess no?

Reply:
I think it may be re-written like debug Attribute that could be toggled on and off dynamically. There already are places in Urho that behave like that.